platform-controller/src/main/java/org/hobbit/controller/docker/ClusterManagerImpl.java

```diff
@@ -40,24 +45,58 @@ public ClusterManagerImpl() throws DockerCertificateException {
         }
     }
 
-    public Info getClusterInfo() throws DockerException, InterruptedException {
-        return dockerClient.info();
+    public ClusterInfo getClusterInfo(){
+        Info info = null;
+        try {
+            info = dockerClient.info();
+        } catch (DockerException e) {
+            LOGGER.error(e.getMessage(), e);
+        } catch (InterruptedException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+
+        ClusterInfo clusterInfo = new ClusterInfo(info.architecture(), info.clusterStore(), info.cgroupDriver(), info.containers(), info.containersRunning(),
+            info.containersStopped(), info.containersPaused(), info.cpuCfsPeriod(), info.cpuCfsQuota(), info.debug(), info.dockerRootDir(), info.storageDriver(),
+            info.driverStatus(), info.experimentalBuild(), info.httpProxy(), info.httpsProxy(), info.id(), info.ipv4Forwarding(),
+            info.images(), info.indexServerAddress(), info.initPath(), info.initSha1(), info.kernelMemory(), info.kernelVersion(), info.labels(), info.memTotal(),
+            info.memoryLimit(), info.cpus(), info.eventsListener(), info.fileDescriptors(), info.goroutines(), info.name(), info.noProxy(), info.oomKillDisable(), info.operatingSystem(),
+            info.osType(), info.systemStatus(), info.systemTime());
```

Comment:
A lot of these methods can not work since you removed the ability to throw exceptions. The problem is that these methods are simply not able to handle the exceptions in a meaningful way (i.e., apart from logging them). The only effect that it will have (in case an error occurs) is that you will cause a `NullPointerException`.

The reason why all the methods in this class throw an exception is to let the piece of code that called a method know that there is a problem. So please add the possibility to throw Exceptions and make use of it.